modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/handlers/MinimumActiveTxTimeCommandHandler.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.table.distributed.raft.handlers;
+
+import java.io.Serializable;
+import org.apache.ignite.internal.lang.IgniteBiTuple;
+import org.apache.ignite.internal.partition.replicator.network.command.UpdateMinimumActiveTxBeginTimeCommand;
+import org.apache.ignite.internal.partition.replicator.raft.snapshot.PartitionDataStorage;
+import org.apache.ignite.internal.replicator.TablePartitionId;
+import org.apache.ignite.internal.table.distributed.raft.MinimumRequiredTimeCollectorService;
+
+/**
+ * RAFT command handler that process {@link UpdateMinimumActiveTxBeginTimeCommand} commands.
+ */
+public class MinimumActiveTxTimeCommandHandler {
+    /** Data storage to which the command will be applied. */
+    private final PartitionDataStorage storage;
+
+    /** Table partition identifier. */
+    private final TablePartitionId tablePartitionId;
```

Comment:
It's a bit tricky. After switching towards colocation track we no longer will have table partition id because there won't be such thing as table partition. Please add a comment explaining that we are not going to substitute TablePartitionId here with ZonePartitionId because we don't won't to rework compaction engine internals.

Reply:
Yep, this definitely makes sense to me. Will add a comment.